pages/components/Header.js

```diff
@@ -1,32 +1,22 @@
 import Head from 'next/head';
 import styled from 'styled-components';
+import { AppBar, Toolbar, Typography } from '@mui/material';
+import Menu from './menu';
 
 export default function Header() {
   return (
     <>
       <Head>
         <title>PARKBENCH BAGELS</title>
       </Head>
-      <Heading>
-        <h1>PARKBENCH BAGELS</h1>
-      </Heading>
+      <AppBar position="static">
+        <Toolbar>
+          <Menu />
```

Comment:
your component is called `BasicMenu`, not `Menu`
```suggestion
          <BasicMenu />
```

remember to update the import accordingly